.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Build WASM and Test
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - run: cd wasmtupelo && docker-compose up -d
```

Comment:
I'm not sure if this is possible, but sometimes I find it helpful to put these longer series of commands into a `.sh` file - i personally think it makes it easier to manage and you get the syntax highlighting. This is still relatively small though and may not be possible